module/x/gravity/types/msgs_test.go

```diff
@@ -73,3 +81,20 @@ func TestValidateMsgDelegateKeys(t *testing.T) {
 	}
 
 }
+
+func VerifyAddressFormat(bz []byte) error {
```

Comment:
Does this function need to be duplicated from module/app/app.go?

Reply:
I wonder if the SDK exposes this function at all

Reply:
So here is the deal.

The SDK's default verifier no longer errors on short addresses. This test's behavior expects errors on short addresses.

SDK address verifiers are global functions so the proper place to put the verifier is app.go and not in the module.

it's not entirely clear what the best way to deal with this change in the SDK behavior is.